modules/rostests/apitests/ntdll/NtQueryInformationToken.c

```diff
@@ -617,6 +617,59 @@ QueryTokenSessionIdTests(
     ok(SessionId == 0, "The session ID of current token must be 0 (current session %lu)!\n", SessionId);
 }
 
+static
+VOID
+QueryTokenIsSandboxInert(
+    _In_ HANDLE Token)
+{
+    NTSTATUS Status;
+    ULONG IsTokenInert;
+    ULONG BufferLength;
+    HANDLE FilteredToken;
+
+    /*
+     * Query the sandbox inert token information,
+     * it must not be inert.
+     */
+    Status = NtQueryInformationToken(Token,
+                                     TokenSandBoxInert,
+                                     &IsTokenInert,
+                                     sizeof(ULONG),
+                                     &BufferLength);
+    ok_ntstatus(Status, STATUS_SUCCESS);
+    ok(IsTokenInert == FALSE, "The token mustn't be a sandbox inert one!\n");
+
+    /*
+     * Try to turn the token into an inert
+     * one by filtering it.
+     */
+    Status = NtFilterToken(Token,
+                           SANDBOX_INERT,
+                           NULL,
+                           NULL,
+                           NULL,
+                           &FilteredToken);
+    if (!NT_SUCCESS(Status))
+    {
+        ok(0, "Failed to filter the current token (Status code %lx)!\n", Status);
```

Comment:
```suggestion
        ok(FALSE, "Failed to filter the current token (Status code %lx)!\n", Status);
```
or use skip() ?

Reply:
`skip()` is the first thing that came up in my mind for such stuff but someone else suggested me to rather fail than skip the test with `ok(0, ...)` when I wrote the tests... :trollface:

Reply:
yes I may understand the logic there: skip() would be more appropriate in the cases of a test to be skipped because e.g. an API is missing in ROS/Windows version being tested, or as a temporary skip until the test is fixed.
While ok(FALSE) would be more like for something that should never fail.